Makefile.cflags

```diff
@@ -3,10 +3,10 @@
 # Otherwise test if the compiler unterstands the "-std=c99" flag, and use it if so.
 ifeq ($(filter -std=%,$(CFLAGS)),)
   ifeq ($(shell $(CC) -std=gnu99 -E - 2>/dev/null >/dev/null </dev/null ; echo $$?),0)
-    CFLAGS += -std=gnu99
+    CFLAGS += -std=gnu11
   else
     ifeq ($(shell $(CC) -std=c99 -E - 2>/dev/null >/dev/null </dev/null ; echo $$?),0)
-      CFLAGS += -std=c99
+      CFLAGS += -std=c11
```

Comment:
Ah, sorry for the noise... But up until this is resolved I would prefer the portable version at #3812.